src/OctoshiftCLI.IntegrationTests/BbsToGithub.cs

```diff
@@ -54,35 +53,46 @@ public BbsToGithub(ITestOutputHelper output)
         _versionClient = new HttpClient();
 
         _sourceBbsHttpClient = new HttpClient();
-        _sourceBbsClient = new BbsClient(logger, _sourceBbsHttpClient, new VersionChecker(_versionClient, logger), new RetryPolicy(logger), sourceBbsUsername, sourceBbsPassword);
-        _sourceBbsApi = new BbsApi(_sourceBbsClient, BBS_URL, logger);
+        _sourceBbsClient = new BbsClient(_logger, _sourceBbsHttpClient, new VersionChecker(_versionClient, _logger), new RetryPolicy(_logger), sourceBbsUsername, sourceBbsPassword);
 
         _targetGithubHttpClient = new HttpClient();
-        _targetGithubClient = new GithubClient(logger, _targetGithubHttpClient, new VersionChecker(_versionClient, logger), new RetryPolicy(logger), new DateTimeProvider(), targetGithubToken);
-        _targetGithubApi = new GithubApi(_targetGithubClient, "https://api.github.com", new RetryPolicy(logger));
+        _targetGithubClient = new GithubClient(_logger, _targetGithubHttpClient, new VersionChecker(_versionClient, _logger), new RetryPolicy(_logger), new DateTimeProvider(), targetGithubToken);
+        _targetGithubApi = new GithubApi(_targetGithubClient, "https://api.github.com", new RetryPolicy(_logger));
 
         _blobServiceClient = new BlobServiceClient(azureStorageConnectionString);
 
-        _sourceHelper = new TestHelper(_output, _sourceBbsApi, _sourceBbsClient, BBS_URL);
         _targetHelper = new TestHelper(_output, _targetGithubApi, _targetGithubClient, _blobServiceClient);
     }
 
-    [Fact]
-    public async Task Basic()
+    [Theory]
+    [InlineData("http://e2e-bbs-8-5-0-linux-2204.eastus.cloudapp.azure.com:7990")]
```

Comment:
Can we add an issue to make the server use HTTPS?  We should probably rotate the credentials as part of this effort, too :wink: